scipy/stats/tests/test_distributions.py

```diff
@@ -971,6 +971,109 @@ def test_stats(self):
         assert_allclose(result, expected)
 
 
+class TestKSTwo(object):
+    def setup_method(self):
+        np.random.seed(1234)
+
+    def test_cdf(self):
```

Comment:
suggest to add a comment: test values for x close to 0 and 1 (exact values given by eq, 1 in Simard / L'Ecuyer);
for x = 0.5: I guess it is related to this identity
  Prob(D_n >= d) = 2*Prob(D_n+ >= d) - Prob(D_n+ >= d and D_n- >= d).
  For d > 0.5, the latter intersection probability is 0.

Reply:
Yes.  Added some doc.